src/components/feedback.tsx

```diff
@@ -72,19 +72,6 @@ export const Feedback: React.FC<{
   const { t } = useTranslation()
   const [submitted, setSubmitted] = useState(null)
 
-  const track = eventType =>
-    analytics.track(eventType, { id }, { context: { ip: '0.0.0.0' } })
-
-  const trackThumbsUp = () => {
-    track('thumbs_up')
-    setSubmitted('thumbs_up')
-  }
-
-  const trackThumbsDown = () => {
-    track('thumbs_down')
-    setSubmitted('thumbs_down')
-  }
-
```

Comment:
I see nice catch

Reply:
Done

Reply:
By removing the 'setSubmitted' state, we won't show feedback anymore when people rate answers. If that's acceptable then LGTM, otherwise we might wanna re-add the `setSubmitted` calls :)